DEVELOPMENT.md

```diff
@@ -0,0 +1,82 @@
+
+## Useful Information
+
+### Models/Apps
+
+The GitHub repository is split into two main parts, Pipelines and ACM. Working on one part usually doesn’t involve working with the other, so you usually won’t have to set-up both dev environments.
+
+One aspect that is similar across both are the *models*/*apps*:
+
+* *Models* (used by Pipelines): These are the actual trained model files generated by Python ([`tf2model/`](https://www.tensorflow.org/guide/checkpoint), [`model.pkl`](https://scikit-learn.org/stable/model_persistence.html), etc.)
+* *Apps* (used by ACM): TL;DR: Sets of `YAML` files that reference images built using Pipelines and uploaded to Quay. See [RHACM Managing Applications](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/applications/managing-applications).
+
+We are currently using two sets of these as examples:
+
+> **Note**
+>
+> 1. [Example TensorFlow model from the MLflow GitHub repository](https://github.com/mlflow/mlflow/tree/ef0f922ca874dd94f9b88e8bfbde2a8cb2c385d2/examples/tensorflow)
+>
+> 2. [Example AzureML + MLflow + sklearn model provided by](https://github.com/stefan-bergstein/azureml-model-to-edge/tree/8e41fbfb2991cf437a6477553939e04c71103555/models/bike-rentals-auto-ml) [@stefan-bergstein](https://github.com/stefan-bergstein)
+>
+> See [#25](https://github.com/opendatahub-io/ai-edge/issues/25) and [#27](https://github.com/opendatahub-io/ai-edge/issues/27) for why.
+
+|                       | **1**                    | **2**                  |
+|-----------------------|--------------------------|------------------------|
+| **Model (Pipelines)** | `tensorflow-housing`     | `bike-rentals-auto-ml` |
+| **App (ACM)**         | `tensorflow-housing-app` | `bike-rental-app`      |
+
+### Clusters
+
+> **Note**
+> See *"Infrastructure Configuration"* in [README.md](./README.md) for more information.
+
+You can visualize the clusters like this:
+
+* `local-cluster` (default): This is ACM hub cluster which represents a fully-fledged OpenShift cluster that's hosted on a cloud provider or at a customer data center. 
+
+* `near-edge-*`:  For an example use case, these are the near-edge clusters that can be located in a server room on-site (e.g. a factory). Typically customers have firewall restrictions for inbound traffic for such clusters and there is a high emphasis on security.
+
+We use a mix of AWS and GCP to host these clusters for diversity. Typically you would use the `local-cluster` to test your work and this includes testing that ACM on the hub will correctly propagate the apps to the `near-edge-*` clusters.
+
+## Pipelines
+
+> **Note**
+> If you’re not going to be pushing new images to Quay, you can use the existing ones from the [`rhoai-edge` Quay organization](https://quay.io/organization/rhoai-edge).
```

Comment:
So should the read (and be setup in such a way) that by default, these are the images that will be used?

Should this go to https://github.com/opendatahub-io/ai-edge/blob/main/pipelines/README.md#quay-repository-and-robot-permissions instead?

Reply:
Upon rereading this note is misplaced and unnecessarily confusing, I'll remove it.